### ISSUE TITLE
Create some sort of Visio Diagram of this environment.

### ISSUE THREAD

**Issue by CCOSTAN** [CCOSTAN/Home-AssistantConfig#7]
Need to be able to visualize this stuff a bit.

**Comment by Chariyski** [comment]
Maybe that will help :) https://www.youtube.com/watch?v=Um_qQJVHlfY&t=7s

**Comment by brianjking** [comment]
@CCOSTAN -- Have you seen [this tool](https://community.home-assistant.io/t/graph-your-configuration/11092) to graph your config?

**Comment by CCOSTAN** [comment]
@brianjking Yeah.. #34   - I can't wait to get it going. Do you have it running?  BTW: Awesome Avatar!

**Comment by CCOSTAN** [comment]
totally not needed anymore.  using floorplan. 

![image](https://user-images.githubusercontent.com/2160436/27397311-ddfb9316-5683-11e7-9bd2-39aaae670260.png)

**Comment by CCOSTAN** [comment]
![bearstoneflow](https://user-images.githubusercontent.com/2160436/43172473-769617e4-8f7d-11e8-8254-72c667940813.png)

**Comment by JamesMcCarthy79** [comment]
Holy Shit dude that is some setup

Regards,

[cid:image001.png@01D36CEA.6DA36C10] [cid:image002.png@01D423FF.DF543F20]

James McCarthy | RF Engineering Consultant | IoT/IoE Solutions Arcitech | Agriculture, Industry 4.0 & Smart City
Powertec Telecommunications Pty Ltd  |  16/511 Olsen Ave, Southport Qld 4215 Australia
P  +61 7 5577 0500  |  E  james.mccarthy@powertec.com.au<mailto: james.mccarthy@powertec.com.au> |  W  www.powertec.io<http://www.powertecwifi.com.au/> |  www.myinsight.io

[linked-in]<https://www.linkedin.com/company/powertec-telecommunications>[facebook]<https://www.facebook.com/powertec.telecom?ref=hl> [Cel-Fi Mini Logo]

**Comment by CCOSTAN** [comment]
Thanks @JamesMcCarthy79 !  I have all the icons available for download to jump start everyone else's diagrams! :) 

https://www.vcloudinfo.com/2018/07/the-bear-stone-home-assistant-icon.html

**Comment by CCOSTAN** [comment]
Yeah .. for every piece of automation or gear, there is a few hours of documentation. :) That's how I keep up! haha.  I've had to force myself to keep gear on the desk until the documentation is caught up.   In Fact, I have to disrupt the diagram already and add in my dash buttons and Dash Wand! haha

**Comment by TUISTERa** [comment]
Nice house btw :)

**Comment by CCOSTAN** [comment]
I probably need to update this again.. Things have really deviated from this diagram.  I think I also want an architectural online this as well. 

![image](https://user-images.githubusercontent.com/2160436/183092563-a7156187-2af0-4c96-9cf0-4db13143f74d.png)
